### ISSUE TITLE
Preserve comments when encoding a decoded and edited config

### ISSUE THREAD

**Issue by khimaros** [lytics/confl#11]
confl is machine editable and human readable which makes it an extremely useful tool. However, using a machine to edit a confl config results in the comments being lost.

**Comment by araddon** [comment]
Yes, this is a great point.  I will think about this, possibly a convention based approach like reflection looking for a "Description" or "Comment" field on the current context?
